lua/luasnip/config.lua

```diff
@@ -120,14 +120,19 @@ local defaults = {
 	load_ft_func = ft_functions.from_filetype_load,
 	-- globals injected into luasnippet-files.
 	snip_env = {
+		ls = require("luasnip"),
```

Comment:
Oh, I think this one is currently leading to errors (check CI-output) due to unresolvable includes ("luasnip" depends on "luasnip.config" depends on "luasnip" ...)
That's a bit unfortunate, I don't think there's any way around this, so it'll have to stay as an exception :/

Reply:
ah, ok. I was wondering why I was getting an error there

Reply:
Oh, sorry, meant exception as "this is the only variable in the doc but not in `snip_env`", not "this exception should stay" xD

Reply:
Oh, my bad. I'll remove it